File: python/7 kyu/190.py
# https://www.codewars.com/kata/562d8d4c434582007300004e/train/python

# Passed

def obfuscate(email):
    char_to_substitute_dict = {
                        "@": " [at] ",
                        ".": " [dot] "
                            }
    
    for char in char_to_substitute_dict:
        substitute = char_to_substitute_dict[char]
        email = email.replace(char, substitute)
        
    return email

output = obfuscate('user_name@example.com')
print(output)

File: nodejs/7 kyu/190.js
// https://www.codewars.com/kata/5506b230a11c0aeab3000c1f/train/javascript

// Passed

function evaporator(content, evapPerDay, threshold) {
  let currentPercent = 100;
  let day = 1;
  while (((100 - evapPerDay) / 100) * currentPercent > threshold) {
    day++;
    currentPercent = ((100 - evapPerDay) / 100) * currentPercent;
  }

  return day;
}

const output = evaporator(100, 5, 5);
console.log(output);